scripts/config.ts

```diff
@@ -378,7 +383,7 @@ function writeConfigs(argv: any) {
   } else {
     let validatorConfig = JSON.parse(baseConfJSON);
     validatorConfig.node.staker.enable = true;
-    validatorConfig.node.staker["use-smart-contract-wallet"] = true;
+    validatorConfig.node.staker["use-smart-contract-wallet"] = false;  // TODO: set to true when fixed
```

Comment:
Updates from upstream. BoLD staker seems to incorrectly with the smart contract wallet